scripts/pcvGuardianDeploy.ts

```diff
@@ -0,0 +1,77 @@
+import hre, { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { utils } from 'ethers';
+import { keccak256 } from 'ethers/lib/utils';
+import config from './config';
+import { Core, PCVGuardian } from '@custom-types/contracts';
+
+const { CORE, PROTOCOL_MULTISIG_ADDRESS, VOLT_FUSE_PCV_DEPOSIT, PCV_DEPOSIT, PRICE_BOUND_PSM } = config;
+
+const PCV_GUARD_ROLE = keccak256(utils.toUtf8Bytes('PCV_GUARD_ROLE'));
```

Comment:
Cleaner to just do `ethers.utils.id('PCV_GUARD_ROLE')` here imo - that will take the keccak256 hash